k8s/benchmark-partitioned.yaml

```diff
@@ -0,0 +1,75 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: camunda-8-benchmark-partitioned
+  labels:
+    app: camunda-8-benchmark-partitioned
+spec:
+  serviceName: camunda-8-benchmark-partitioned
+  selector:
+    matchLabels:
+      app: camunda-8-benchmark-partitioned
+  replicas: 3
+  template:
+    metadata:
+      labels:
+        app: camunda-8-benchmark-partitioned
+      annotations:
+        prometheus.io/scrape: "true"
+        prometheus.io/port: "8088"
+        prometheus.io/path: "/actuator/prometheus"
+    spec:
+      containers:
+        - name: camunda-8-benchmark
+          image: camundacommunityhub/camunda-8-benchmark:main
+          imagePullPolicy: Always
+          env:
+            - name: POD_NAME
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.name
```

Comment:
Please remove this and all references to Kubernetes concepts from the Java code. The application must also be able to work without Kubernetes.